packages/backend/src/api/__tests__/factory.test.ts

```diff
@@ -88,39 +86,6 @@ export default (QUnit: QUnit) => {
       );
     });
 
-    test('executes a successful backend API request for a resource that contains data (key related to pagination)', async assert => {
```

Comment:
🔧  Instead of removing this test, replace the `apiClient.emails.createEmail(body);` with another API call. This test is not related to the `EmailAPI`, it was just used as an example.

Reply:
Is it possible this test is no longer needed? 🤔 After looking around only `SMSMessage` and `Emails` contain the property `readonly data?: Record<string, any> | null` that is being checked here.

To be completely honest I am not 100% sure what is being tested here  😓  And whats the differences between this test and the first one that `executes a successful backend API request for a single resource and parses the response`

Reply:
You are correct. This case is no longer applicable. let's drop it.